packages/sanity/src/structure/structureBuilder/DocumentList.ts

```diff
@@ -142,7 +142,7 @@ export class DocumentListBuilder extends GenericListBuilder<
   }
 
   /** Set Document list filter
-   * @param filter - filter
+   * @param filter - GROQ-filter used to determine which documents to display. Do not support joins, since they operate on individual documents, and will ignore order-clauses and projections. See https://www.sanity.io/docs/realtime-updates
```

Comment:
Use `{@link <MemberInAPI|external qualified url>}` to create a link

```suggestion
   * @param filter - GROQ-filter used to determine which documents to display. Do not support joins, since they operate on individual documents, and will ignore order-clauses and projections. See {@link https://www.sanity.io/docs/realtime-updates}
```